<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctedit</class>
 <widget class="QDialog" name="ctedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cross Table Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="fraBottoni">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="cmdHideShow">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdButt2">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fraEdit">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="fraOptions">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="optRetentive">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Retentive</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="optNonRetentive">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Non Retentive</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblPriority">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Priority</string>
            </property>
            <property name="buddy">
             <cstring>cboPriority</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblUpdate">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
            <property name="buddy">
             <cstring>cboUpdate</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lblName">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
            <property name="buddy">
             <cstring>txtName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="lblType">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Type</string>
            </property>
            <property name="buddy">
             <cstring>cboType</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="lblDecimal">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Decimal</string>
            </property>
            <property name="buddy">
             <cstring>cboDecimal</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="lblProtocol">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Protocol</string>
            </property>
            <property name="buddy">
             <cstring>cboProtocol</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="lblIp">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>IP Address</string>
            </property>
            <property name="buddy">
             <cstring>txtIP</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLabel" name="lblPort">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Port</string>
            </property>
            <property name="buddy">
             <cstring>txtPort</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QLabel" name="lblNode">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Node ID</string>
            </property>
            <property name="buddy">
             <cstring>txtNode</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QLabel" name="lblRegister">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Register</string>
            </property>
            <property name="buddy">
             <cstring>txtRegister</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="10">
           <widget class="QLabel" name="lblBlock">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Block</string>
            </property>
            <property name="buddy">
             <cstring>txtBlock</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="11">
           <widget class="QLabel" name="lblBlockSize">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Block Size</string>
            </property>
            <property name="buddy">
             <cstring>txtBlockSize</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="cboPriority">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cboUpdate">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="txtName">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="cboType">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QComboBox" name="cboDecimal">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="cboProtocol">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLineEdit" name="txtIP">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QLineEdit" name="txtPort">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="8">
           <widget class="QLineEdit" name="txtNode">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="9">
           <widget class="QLineEdit" name="txtRegister">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="10">
           <widget class="QLineEdit" name="txtBlock">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="11">
           <widget class="QLineEdit" name="txtBlockSize">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblComment">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Comment</string>
          </property>
          <property name="buddy">
           <cstring>txtComment</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtComment">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblCT"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
